gce/gce.go

```diff
@@ -688,6 +694,45 @@ func (s *gceOps) SetInstanceGroupSize(instanceGroupID, instanceGroupLocation str
 	return nil
 }
 
+func (s *gceOps) GetInstanceGroupSize(instanceGroupName string) (int64, error) {
+
+	zonalCluster, err := isZonalCluster(s.inst.clusterLocation)
+	if err != nil {
+		return 0, err
+	}
+
+	var nodePool *container.NodePool
+	if zonalCluster {
+		nodePool, err = s.containerService.Projects.Zones.Clusters.NodePools.Get(
+			s.inst.project, s.inst.clusterLocation, s.inst.clusterName, instanceGroupName).Do()
+	} else {
+		nodePoolPath := fmt.Sprintf("projects/%s/locations/%s/clusters/%s/nodePools/%s",
+			s.inst.project, s.inst.clusterLocation, s.inst.clusterLocation, instanceGroupName)
+		nodePool, err = s.containerService.Projects.Locations.Clusters.NodePools.Get(nodePoolPath).Do()
+	}
+
+	if err != nil {
+		return 0, err
+	}
+
+	nodeCount := int64(0)
+	for _, instanceGroupURL := range nodePool.InstanceGroupUrls {
+
+		nodeGrpName := strings.TrimSpace(filepath.Base(instanceGroupURL))
+
+		temp := strings.SplitAfter(instanceGroupURL, "zones")[1]
+		zone := strings.Split(temp, "/")[1]
```

Comment:
https://golang.org/pkg/path/#Base might be a better fit here instead of split and indexing.

Reply:
Hi @harsh-px ,

https://golang.org/pkg/path/#Base will be useful while getting last element of the URL. Hence used it here to extract `nodeGrpName`

```suggestion
		nodeGrpName := strings.TrimSpace(filepath.Base(instanceGroupURL))
```

But for extracting zone from bellow url, I am not sure we can use it.
https://www.googleapis.com/compute/v1/projects/portworx-eng/zones/us-central1-a/instanceGroupManagers/gke-torpedo-tp-2-1-2-gke-default-pool-fbff8a1b-grp